src/main/java/com/fasterxml/jackson/core/io/ContentReference.java

```diff
@@ -98,6 +110,28 @@ protected ContentReference(boolean isContentTextual, Object rawContent,
         _rawContent = rawContent;
         _offset = offset;
         _length = length;
+        ErrorReportConfiguration errorReportConfiguration = ErrorReportConfiguration.defaults();
```

Comment:
Instead of this assignment, I think constructor should just call new one, with extra argument specified as `ErrorReportConfiguration.defaults()`.